server/src/test/scala/org/http4s/server/middleware/authentication/AuthMiddlewareSpec.scala

```diff
@@ -0,0 +1,48 @@
+package org.http4s.server.middleware.authentication
+
+import cats.data.Kleisli
+import fs2.Task
+import org.http4s.server.AuthMiddleware
+import org.http4s._
+import org.http4s.dsl._
+
+class AuthMiddlewareSpec extends Http4sSpec {
+
+  "AuthMiddleware" should {
+    "work" in {
```

Comment:
This is implemented in master in this location. So I'm ok with it being here for the time being, unless @rossabaker would like to take this moment to make a change.